pom.xml

```diff
@@ -53,8 +53,13 @@
         <sonar.organization>green-code-initiative</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
-        <sonarqube.version>9.4.0.54424</sonarqube.version>
-        <sonarjava.version>7.19.0.31550</sonarjava.version>
+        <!-- Version of SonarQube for integration tests -->
+        <sonarqube.version>9.9.7.96285</sonarqube.version>
```

Comment:
ok for me to upgrade library versions but, can you check the minimum sonarqube version required for these upgrades and fill it in changelog.md file and readme.md compatibility array, please ?

Reply:
I took these versions from SonarQube 9.x LTS download.

I have updated [compatibility matrix](https://github.com/green-code-initiative/ecoCode-java/pull/72/commits/61e760386696de3bf7fc0a172a6c65452ca731e7) (I assumed that the next version of the plugin would be v2.0)